packages/2d/src/Components/LowLevelMouse.ts

```diff
@@ -160,6 +162,14 @@ export default function LowLevelMouse() {
     };
   };
 
+  const handleMouseOut = () => {
+    storage.outCallbacks.forEach((callback) => callback(event));
```

Comment:
We should call `updateEvent` in these handlers, too, so that the clientX/clientY data is up-to-date.